app/views/slices/gallery.php

```diff
@@ -4,25 +4,27 @@
 ?>
 
 <section class="gallery content-section">
-  
-  <?php
-  // Loop through the gallery items
-  foreach ( $slice->items as $galleryItem ) {
-  ?>
-  
-  <div class="gallery-item">
-    <img src="<?= $galleryItem->image->url ?>"/>
-    <?= RichText::asHtml($galleryItem->image_description, $prismic->linkResolver); ?>
-
+  <?= RichText::asHtml($slice->primary->gallery_title); ?>
+  <div class="gallery">
```

Comment:
You call the class "gallery" on this and it's parent. This can be done cleaner, I think. I'm guessing that you just need to remove the class from the parent div.

Reply:
It was a confusing leftover from the previous version. Changed it to reflect the CSS designation from the nodeJS example